docs/commands/ft.create.md

```diff
@@ -24,8 +24,18 @@ Create an index with the given specification
     [NOFREQS] 
     [STOPWORDS count [stopword ...]] 
     [SKIPINITIALSCAN]
-    SCHEMA field_name [AS alias] TEXT | TAG | NUMERIC | GEO | VECTOR [ SORTABLE [UNF]] 
-    [NOINDEX] [ field_name [AS alias] TEXT | TAG | NUMERIC | GEO | VECTOR [ SORTABLE [UNF]] [NOINDEX] ...]
+    SCHEMA field_name [AS alias] 
+      TEXT [SORTABLE] [UNF] [NOINDEX] [NOSTEM] [WEIGHT] [PHONETIC] [WITHSUFFIXTRIE] |
+      TAG [SORTABLE] [UNF] [NOINDEX] [SEPARATOR] [CASESENSITIVE] [WITHSUFFIXTRIE] |
+      NUMERIC [SORTABLE] [NOINDEX] |
+      GEO [SORTABLE] [NOINDEX] |
+      VECTOR ... [ SORTABLE [UNF]] 
```

Comment:
We can list basic parameters for VECTOR
`VECTOR {algorithm} {count} [{attribute_name} {attribute_value} ...]`

And use the existing link to the [Vector Fields](/redisearch/reference/vectors) page for further details

Reply:
What about [SORTABLE [UNF]]?

Reply:
Irrelevant for VECTOR